storage/innobase/handler/ha_innodb.cc

```diff
@@ -14782,7 +14782,7 @@ ha_innobase::info_low(
 	char		path[FN_REFLEN];
 	os_file_stat_t	stat_info;
 
-	DBUG_ENTER("info");
+	DBUG_ENTER("ha_innobase::info");
```

Comment:
Why? Do you really use DBUG facility? We also don't normally include unrelated changes in our commits.

Reply:
> Why? Do you really use DBUG facility? We also don't normally include unrelated changes in our commits.

Good point, thanks! We completely agree. We should minimize unnecessary changes to the server code, and keep all modifications contained within `storage/videx` as much as possible.

I think this could just be some scaffolding from @haibo's debugging phase. These changes shouldn't be part of the submission, even in a WIP PR, and will be removed. cc @YoungHypo